buildkite-ec2-stack/scripts/reboot.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -euo pipefail
+
+sudo reboot
```

Comment:
😄 
Yeah it's pretty stupid  
However: Inline scripts in packer just show some random string instead of a human-readable name like "reboot", so there is some value in using this awesome script